crmsh/utils.py

```diff
@@ -679,6 +679,7 @@ def str2tmp(_str, suffix=".pcmk"):
     if not s.endswith('\n'):
         f.write("\n")
     f.close()
+    os.close(fd)
```

Comment:
CI failed after this line since that file needs to return and continue to be used

Reply:
Removed it.

Even though I don't understand why it'd fail- since I'm just closing the FD and returning the file name.

Reply:
Uh, my bad. Understood.
```
f = os.fdopen(fd, "w")
```
is being used. So, `f.close()` is enough. No need to close the FD manually.